src/plugins/charCounter/index.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import "./styles.css";

```

Comment:
```suggestion
import style from "./styles.css?managed";

```
and use enableStyle/disableStyle from api/Styles in start() stop() so css only applied when plugin is on (since you use discord class name now)